### ISSUE TITLE
Make reactions accessible to screenreaders

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#11089]
> >    (how well do reactions work on a screenreader btw?)

> That's an interesting question, now that I am looking into them. I see that there are reactions on my message. But when I hover over the message with the mouse, I see options to answer and open the More Options menu, but don't see an item that would allow me to react to a message. So, effectively, I see reactions to messages from others, but cannot react myself currently. That's for the web app.

> There's a react button to the left of the reply button that shows when you hover over a message. Hovering over the react button shows a menu containing 8 common emojis one can react with. This menu is a custom popup, which may fail to be picked up by the screenreader.

> Yup, the Add Reaction button is currently only a span with a class, no accessible semantics like the other buttons have.

> Yes, the button itself doesn't show up for screen readers yet. Unlike the Reply button and others. And, it would need to show its menu on press rather than hover. Screen reader users don't use a mouse to interact with elements comonly. This is an extension to my earlier bug report then that things should be activable via keyboard/a defined action rather than just hovering. In essence, this button must react the same way and open a similar popup like the Options button to the right of the Reply button.

**Comment by t3chguy** [comment]
Duplicate of https://github.com/vector-im/riot-web/issues/11082
